src/staging/src/pages/find-a-park.js

```diff
@@ -646,7 +646,7 @@ export default function FindAPark({ location, data }) {
                     </div>
                     <div className="col-12 pr-3">
                       <div className="">
-                        <h4 className="filter-heading p10t">Popular</h4>
+                        <p className="filter-heading p10t">Popular</p>
```

Comment:
I don't think this fix really captures the full intent of the error in the accessibility scan.  The screenreader needs to know that this the header for a form group. 

e,g,
```
<fieldset>
        <legend>I want to receive</legend>
	<div>
		<input type="checkbox" name="newsletter" id="check_1">
                <label for="check_1">The weekly newsletter</label>
	</div>
</fieldset>
```
example from https://www.w3.org/WAI/tutorials/forms/grouping/